From c69d547638f09d0bb99f14c74a1c8be485cf2c72 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms@verge.net.au>
Date: Fri, 1 Nov 2013 16:48:30 +0900
Subject: [PATCH 112/179] CLEANUP: Remove unused 'last_slowstart_change' field
 from struct peer

This was inadvertently added by "MEDIUM: checks: Add agent health check".
It appears to have never been used.

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 include/types/peers.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/types/peers.h b/include/types/peers.h
index 41a1cce..d52dc8c 100644
--- a/include/types/peers.h
+++ b/include/types/peers.h
@@ -72,7 +72,6 @@ struct peer {
 		int line;	  /* line where the section appears */
 	} conf;		  	  /* config information */
 	time_t last_change;
-	time_t last_slowstart_change;
 	struct sockaddr_storage addr;  /* peer address */
 	struct protocol *proto;	       /* peer address protocol */
 	struct xprt_ops *xprt;         /* peer socket operations at transport layer */
-- 
1.8.1.5

